tests/Sdk/Acceptance/Extension/Task/AnalyzerReportTaskCest.php

```diff
@@ -1,50 +0,0 @@
-<?php
```

Comment:
I don't think we need to remove these tests. It's kind of smoke integration tests for us and I believe important because the upgrader is important part of the sdk